### PR TITLE
prepare for v1.5 release

### PR DESCRIPTION
**Pull request by johanneshiry**
Release branch for v1.5

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased/Snapshot]
 
+## [1.5]
```
Comment (ckittl) [inline CHANGELOG.md:9]: The tag `[1.5]` needs to be specified (cf. end of file)

File: src/main/java/edu/ie3/util/StreamUtils.java [hunk 2]
```diff
@@ -18,6 +18,11 @@
  * @since 13.04.19
  */
 public class StreamUtils {
+
+  private StreamUtils() {
+    throw new IllegalStateException("Utility class");
```
Comment (ckittl) [inline src/main/java/edu/ie3/util/StreamUtils.java:23]: Please use a meaningful message. I would suggest `"This is a Utility Class and not meant to be instantiated."` from `FileIoUtils`. Might also be an idea to put this string into a `UtilityHelper`-class or so to have the same message in each utility class.

File: src/main/java/edu/ie3/util/copy/DeepCopy.java [hunk 3]
```diff
@@ -25,20 +25,24 @@
 public class DeepCopy {
   private static final Logger logger = LoggerFactory.getLogger(DeepCopy.class);
 
+  private DeepCopy() {
+    throw new IllegalStateException("Utility class");
```
Comment (ckittl) [inline src/main/java/edu/ie3/util/copy/DeepCopy.java:29]: Same here

**Review by ckittl — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (johanneshiry) [reply CHANGELOG.md:9]: would prefer to do this in the v1.6-snapshot branch as I would like to include the merge commit of this branch into master into the range of the v1.5 commits

File: src/main/java/edu/ie3/util/StreamUtils.java (hunk 2 above)
Comment (johanneshiry) [reply src/main/java/edu/ie3/util/StreamUtils.java:23]: ACK.

File: src/main/java/edu/ie3/util/copy/DeepCopy.java (hunk 3 above)
Comment (johanneshiry) [reply src/main/java/edu/ie3/util/copy/DeepCopy.java:29]: ACK.

**Review by ckittl — APPROVED** [review]

**Pull request merged**